### ISSUE TITLE
Implement remaining dialogs

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sunfall/endgame-singularity#55]
```
The finance screen and concepts dialogs still need to be implemented under
the new GUI system, and the cheat menu still needs its functionality
implemented.
```

Original issue reported on code.google.com by `funnyman3595` on 11 Aug 2008 at 8:39

**Comment by GoogleCodeExporter** [comment]
```
Also need a way to view the progress of a base under construction.
```

Original comment by `funnyman3595` on 17 Aug 2008 at 9:55

**Comment by GoogleCodeExporter** [comment]
```
Finance is working (and has been for some time now), but could use 
beautification. 
Not a blocker for closing this, but it'd be nice to have.
```

Original comment by `funnyman3595` on 26 Sep 2008 at 7:53

**Comment by GoogleCodeExporter** [comment]
```
Also, make sure Escape and RMB are consistently available for backing out of a 
dialog.
```

Original comment by `funnyman3595` on 13 Oct 2008 at 9:49

**Comment by GoogleCodeExporter** [comment]
```
Don't forget about the cheat dialog.
```

Original comment by `evilmrhe...@gmail.com` on 23 Oct 2008 at 3:20

**Comment by GoogleCodeExporter** [comment]
```
Cheat dialog is working again in git (131920) and SVN (r904).  It was already
mentioned in the initial post, but while you had me thinking about it, I just 
went
ahead and fixed it.

Still waiting on the concepts dialogs and Escape/RMB.
```

Original comment by `funnyman3595` on 23 Oct 2008 at 4:51

**Comment by GoogleCodeExporter** [comment]
```
Escape/RMB is now working in git (cb6ff2) and SVN (r906).
```

Original comment by `funnyman3595` on 23 Oct 2008 at 6:11

**Comment by GoogleCodeExporter** [comment]
```
With the recent changes to the Knowledge dialog by emh, isn't this puppy done?
```

Original comment by `phil.bor...@gmail.com` on 22 Jan 2009 at 4:06

**Comment by GoogleCodeExporter** [comment]
```
We still need the "way to view the progress of a base under construction" 
mentioned
earlier, but otherwise this is done.
```

Original comment by `funnyman3595` on 26 Jan 2009 at 10:19
* Changed title: **Progress view for bases under construction**
* Added labels: **Type-Enhancement**, **Priority-High**, **Milestone-Pre1.0**
* Removed labels: **Type-Defect**, **Priority-Critical**, **Milestone-0.29**

**Comment by GoogleCodeExporter** [comment]
```
That portion has been replaced by issue 86, to simplify things a bit.
```

Original comment by `funnyman3595` on 24 May 2009 at 2:18
* Changed title: **Implement remaining dialogs**
* Changed state: **Fixed**
